ENIGMAsystem/SHELL/Makefile

```diff
@@ -59,14 +61,14 @@ CC := gcc
 DEPENDENCIES :=
 
 ifeq ($(GMODE), Debug)
-	override CXXFLAGS += -Wall -g -DDEBUG_MODE
+	override CXXFLAGS += -Wall -std=c++11 -g -DDEBUG_MODE
```

Comment:
I think you should be specifying these in the MacOSX `gcc.ey` rather than directly in the makefile.